### ISSUE TITLE
Log which user enabled expert mode

### ISSUE THREAD

**Issue by arnaudsjs** [inmanta/inmanta-core#5687]
Depends on: https://github.com/inmanta/web-console/issues/4594

**Comment by sanderr** [comment]
Probably best to implement this using ContextVar.

**Comment by sanderr** [comment]
Probably best to implement this using ContextVar.

**Comment by arnaudsjs** [comment]
The JWT tokens created by the Inmanta server currently don't include any information regarding the user where the token belongs to. It would be good to include the user in the same way in the token as Keycloak does it. The integration with Keycloak is currently broken when SSL is enabled on the Inmanta server (See: inmanta/web-console#4594). It was decided together with Bart to postpone the implementation of this ticket until the integration with Keycloak is fully working and we have a better view regarding the structure of the payload of the tokens created by Keycloak.
